lib/eksek.rb

```diff
@@ -8,11 +8,11 @@
 module Kernel
   private
 
-  def eksek cmd, opts = {}, &block
-    Eksekuter.new(cmd, opts).run(&block)
+  def eksek *args, **opts, &block
```

Comment:
Would it add any value if the `Eksekuter` would check the sanity of the args?

Reply:
What kind of check do you have in mind? Type checks?

Reply:
Yes I had first. However, this might be superfluous since we pass stuff on to Open3 anyway... So feel free to not do it and merge. 🤗